paimon-flink/paimon-flink-cdc/src/main/java/org/apache/flink/table/catalog/CatalogMaterializedTable.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.catalog;
+
+/**
+ * Dummy placeholder to resolve compatibility issue of CatalogMaterializedTable(introduced in flink
+ * 1.20).
+ */
+public interface CatalogMaterializedTable extends CatalogBaseTable {
```

Comment:
Paimon-Flink 1.20 will shade paimon-flink-cdc too.

Can you upgrade flink version of paimon-flink-cdc first?

Reply:
I manage to update it to `1.20.0`, but CI pipeline raise `ClassNotFoundException`. I didn't go any further, but I think we might be able to upgrade this in other PR, WDYT?